infrastructure/logging.tf

```diff
@@ -1,3 +1,29 @@
+# CloudTrail
+
+resource "aws_cloudtrail" "data_refinery_s3_cloudtrail" {
+  name                          = "data-refinery-s3-cloudtrail-${var.user}-${var.stage}"
+  depends_on                    = ["aws_s3_bucket.data_refinery_bucket"]
+  s3_bucket_name                = "${aws_s3_bucket.data_refinery_bucket.id}"
```

Comment:
This bucket is where we store our processed data. Could you create a new bucket for storing these logs?